caampr-backend/build.gradle

```diff
@@ -22,9 +22,15 @@ targetCompatibility = 1.8
 repositories {
     // Use JCenter for resolving dependencies.
     jcenter()
+    mavenCentral()
 }
 
 dependencies {
+    implementation platform('com.amazonaws:aws-java-sdk-bom:1.11.969')
```

Comment:
what is this `platform` doing? why can't we just bring in the the dynamodb client/model dependencies in the implementation section, like I did with the lambda stuff?

Reply:
required for gradle build, from what i can gather from docs